billy/web/api/handlers.py

```diff
@@ -490,7 +485,7 @@ def _ocd_id_to_shape_url(self, ocd_id):
         )
         data = json.load(urllib2.urlopen(url))
         geometries = filter(
-            lambda x: x['boundary_set']['name'] in ACTIVE_BOUNDARY_SETS,
+            lambda x: re.search(ACTIVE_BOUNDARY_SETS_RE, x['boundary_set']['name']),
```

Comment:
The `settings.py`'s RE already accounts for extra text afterwards, so that's all good, right?